File: JAVA/Java_notes/File_Handling/Java/CreateFile.java
package File_Handling.Java;
import java.io.*;
public class CreateFile {
    public static void main(String[] args) throws Exception {
        File fobj = new File("Infosystem.txt");

        if(fobj.createNewFile()) 
        {
            System.out.println("File Successfully created");
        }
        else
        {
            System.out.println("Unable to create a file");
        }
    }
}

File: JAVA/Java_notes/File_Handling/C/CreateFile.c
#include <stdio.h>
#include <unistd.h> // Universal Standard
#include <fcntl.h> // File Control
#include <stdlib.h> 

int main () 
{
 
   char Name[30];
   int FD = 0; // File Descripters

   printf("Enter file name that you want to create: ");
   scanf("%s", Name);

   FD = creat(Name,0777);
   if (FD == -1)
   {
    printf("Unable to create file");
   }
   else
   {
    printf("File created successfully\n");
   }
 return 0;
}
